actions/setup-crib-environment/action.yml

```diff
@@ -0,0 +1,169 @@
+name: "Setup CRIB Environment"
+description:
+  "Composite action for deploying a CRIB, setting up GAP, Nix, and deploying to
+  an ephemeral environment."
+
+inputs:
+  api-gateway-host:
+    description: API Gateway host for K8s
+    required: true
+  aws-region:
+    description: AWS region
+    required: true
+  aws-role-arn:
+    description: AWS Role ARN for CRIB
+    required: true
+  devspace-ingress-cidrs:
+    default: 0.0.0.0/0
+    description: DevSpace ingress CIDRs
+    required: false
+  disable-environment-teardown:
+    default: "false"
+    description: Disable namespace deletion for debugging purposes.
+    required: false
+  ecr-private-registry:
+    default: ""
+    description: ECR private registry account ID
+    required: false
+  ecr-private-registry-stage:
+    default: ""
+    description: ECR private registry account ID in Staging
+    required: false
```

Comment:
these 2 seems to be duplicated, the first one is not used in the workflow

Reply:
I think it's used by the proxy image inside the GAP action, right? That's the reason I have added it.